phpmyfaq/src/phpMyFAQ/Administration/Backup.php

```
@@ -153,4 +157,33 @@ private function getBackupHeader(string $tableNames): array
             '-- Otherwise this backup will be broken.'
         ];
     }
+
+    public function createContentFolderBackup(): string|bool
+    {
+        $zipFile = PMF_ROOT_DIR . DIRECTORY_SEPARATOR . 'content.zip';
+
+        $zipArchive = new ZipArchive();
+        if ($zipArchive->open($zipFile, ZipArchive::CREATE) !== true) {
+            return false;
```

We should throw an exception instead of just `false`.